src/frontend/src/eth/services/send.services.ts

```diff
@@ -169,6 +169,29 @@ const ckErc20HelperContractPrepareTransaction = async ({
 	});
 };
 
+/**
+ * Get the current allowance of an Erc20 contract.
+ */
+// TODO: this function is still not used in the codebase, so for now we export globally
```

Comment:
Maybe an eslint exception instead of exposing it if not used?

Reply:
better!